include/Foundation/NSFileManager.h

```diff
@@ -18,6 +18,7 @@
 #import <Foundation/FoundationExport.h>
 #import <Foundation/NSObject.h>
 #import <Foundation/NSPathUtilities.h>
+#import <Foundation/NSDictionary.h>
```

Comment:
Why is libclang choking when this isn't here?
Is this new?
Presumably this file builds in build.sln. 
So, it is going to be very hard to know when apianalyzer is going to break #import stuff vs. @class